api/MutationObserver.json

```diff
@@ -0,0 +1,473 @@
+              },
+              {
+                "version_added": "7"
+              }
+            ],
+            "safari_ios": [
+              {
+                "version_added": "6",
+                "version_removed": "7",
+                "notes": "Prefixed with <code>-webkit</code>"
+              },
+              {
+                "version_added": "7"
+              }
+            ],
+            "samsunginternet_android": {
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": true
+          }
+        }
+      },
+      "observe": {
+        "__compat": {
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": [
+              {
+                "version_added": "18",
+                "version_removed": "26",
+                "notes": "Prefixed with <code>-webkit</code>."
+              },
+              {
+                "version_added": "26"
+              }
+            ],
+            "chrome_android": [
+              {
+                "version_added": "18",
+                "version_removed": "26",
+                "notes": "Prefixed with <code>-webkit</code>."
+              },
+              {
+                "version_added": "26"
+              }
+            ],
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "14"
+            },
+            "firefox_android": {
+              "version_added": "14"
+            },
+            "ie": {
+              "version_added": "11"
+            },
+            "opera": {
+              "version_added": "15"
+            },
+            "opera_android": {
+              "version_added": "15"
+            },
+            "safari": [
+              {
+                "version_added": "6",
+                "version_removed": "7",
+                "notes": "Prefixed with <code>-webkit</code>"
+              },
+              {
+                "version_added": "7"
+              }
+            ],
+            "safari_ios": [
+              {
+                "version_added": "6",
+                "version_removed": "7",
+                "notes": "Prefixed with <code>-webkit</code>"
+              },
+              {
+                "version_added": "7"
+              }
+            ],
+            "samsunginternet_android": {
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": true
+          }
+        }
+      },
+      "disconnect": {
+        "__compat": {
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
```

Comment:
This should be:
```json
"webview_android": {
  "version_added": "18"
}
```
And moved to the end of the `support` block (see #398 and #1882).